src/app/components/Brand/index.jsx

```diff
@@ -5,31 +5,55 @@ import VisuallyHiddenText from '../VisuallyHiddenText';
 import {
   C_POSTBOX,
   C_WHITE,
-  GEL_SPACING_DBL,
+  GEL_SPACING,
+  GEL_SPACING_HLF,
 } from '../../lib/constants/styles';
 import {
   layoutGridWrapper,
   layoutGridItem,
   layoutGridItemFullWidth,
 } from '../../lib/layoutGrid';
 
+const SVG_TOP_OFFSET = '1.25rem'; // 20px
+const SVG_BOTTOM_OFFSET = '1.5rem'; // 24px
+const BANNER_HEIGHT = '5rem'; // 80px
+
+const SVG_HEIGHT_PX = 24;
+const SVG_HEIGHT = `${SVG_HEIGHT_PX / 16}rem`;
```

Comment:
Why isn't this just set to `1.5rem`? If `16` is indicative of a base font-size, we can't use it, as it might not 16.

Same applies to width.

Reply:
While my comment is valid, I do not think it is worth the effort of considering how it might be implemented, If it is an issue, we can fix forward.